android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt

```diff
@@ -411,6 +424,20 @@ fun ListingDetailsPageView(
 	}
 }
 
+@Composable
+fun ImageBoxPlaceholder(
+	modifier: Modifier = Modifier,
+	contentAlignment: Alignment = Alignment.Center,
+	content: @Composable () -> Unit)
```

Comment:
It should be `) {` if you know what I mean.

Reply:
oops my bad, yes